### PR TITLE
J-309: Add User Profile Screen

### PR DESCRIPTION
**Pull request by EugeniusGR**
image
image

### CODE REVIEW

File: src/screens/User/common/types.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  image: string
+}
+
+type User = {
+  firstname: string
+  surname: string
+  nickname: string
+  email: string
+  image: string
+}
```
Comment (eugeneprib) [inline src/screens/User/common/types.ts]: Don't we have types for Podcast and User?

File: src/screens/User/User.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                label={userPageInfo.user.email}
+              />
+            </TouchableOpacity>
+          </View>
+        </View>
+      </View>
+
+      <View style={styles.userDataContainer}>
+        <Heading type={HeadingType.MEDIUM} label="Podcasts" />
+        {userPageInfo.podcasts.map((podcast, inx) => {
```
Comment (eugeneprib) [inline src/screens/User/User.tsx]: Use FlatList instead, move mapper out of this scope

**Review by eugeneprib — CHANGES_REQUESTED** [review]

**Comment by EugeniusGR** [comment]
image

**Comment by EugeniusGR** [comment]
Screenshot_1630499057

File: src/assets/images/atMark.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+</svg>
```
Comment (Denidzio) [inline src/assets/images/atMark.svg]: Let's add newline for .svg

File: src/screens/User/User.tsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+const userPageInfo: Props = {
+  podcasts: MockedPodcasts,
+  user: MockedUser
+}
+
+const UserPage: React.FC<Props> = () => {
+  const handleLogOut = () => {
+    return 0
```
Comment (Denidzio) [inline src/screens/User/User.tsx:24]: What does it mean `0`? Put it to `constants`.

File: src/screens/User/User.tsx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+              <Check width={15} />
+              <PlainText
+                style={styles.userInfoItemText}
+                label={userPageInfo.user.nickname}
+              />
+            </View>
+            <TouchableOpacity
+              activeOpacity={0.7}
+              style={styles.userInfoItem}
+              onPress={() =>
```
Comment (Denidzio) [inline src/screens/User/User.tsx]: Avoid inline functions, please.

File: src/screens/User/User.tsx [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+          </View>
+        </View>
+      </View>
+
+      <View style={styles.userDataContainer}>
+        <Heading type={HeadingType.MEDIUM} label="Podcasts" />
+        <FlatList
+          data={userPageInfo.podcasts}
+          renderItem={renderItem}
+          keyExtractor={(item) => item.name}
```
Comment (Denidzio) [inline src/screens/User/User.tsx:96]: Are you sure that `name` is good unique key?

File: src/screens/User/User.tsx [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+          contentContainerStyle={{ paddingBottom: 160 }}
```
Comment (Denidzio) [inline src/screens/User/User.tsx]: Let's put the style to `styles.ts`

**Review by Denidzio — CHANGES_REQUESTED** [review]

File: src/screens/User/User.tsx (hunk 4 above)
Comment (EugeniusGR) [reply src/screens/User/User.tsx:24]: thats for now, I will do logic here in the next task. Thats jush 0. Nothing special

File: src/screens/User/User.tsx (hunk 6 above)
Comment (EugeniusGR) [reply src/screens/User/User.tsx:96]: I cant add a number value, so thats all I can do, I guess

**Pull request merged**